RebbleOS/vibrate.c

```diff
@@ -48,70 +67,97 @@ void vibrate_init(void)
     _vibrate_queue = xQueueCreate(1, sizeof(uint8_t));
```

Comment:
also could be clearer that the uint8_t datasize is likely wrong now. Are you not passing a 32 bit pointer to the vibrate struct now?